docs/usage/training_time_compression/quantization_aware_training/Usage.md

```diff
@@ -0,0 +1,96 @@
+# Use Neural Network Compression Framework (NNCF) for Quantization Aware Training in PyTorch
+
+This is a step-by-step tutorial on how to integrate the NNCF package into the existing PyTorch project.
+The use case implies that the user already has a training pipeline that reproduces training of the model in the floating  point precision and pretrained model.
+The task is to prepare this model for accelerated inference by simulating the compression at train time.
+Please refer to this [document](/docs/usage/training_time_compression/other_algorithms/LegacyQuantization.md) for quantization implementation details.
```

Comment:
for quantization implementation details => for details of the implementation.

Reply:
Done